src/compressio/compression_algorithms/type_compressions.py

```diff
@@ -21,6 +22,34 @@ def get_compressed_type(
     return next(test_sequence)
 
 
+def compress_sparse(series: pd.Series) -> pd.Series:
+    number = len(series)
+    missing_count = number - series.count()
+
+    if missing_count > 0:
```

Comment:
is this different than `series.hasnans`?

Reply:
We made use of the count before, but now that we don't we're better off with `series.hasnans`